api/nodes/urls.py

```diff
@@ -7,6 +7,7 @@
     # url(r'^blog/', include('blog.urls')),
     url(r'^$', views.NodeList.as_view(), name='node-list'),
     url(r'^(?P<pk>\w+)/$', views.NodeDetail.as_view(), name='node-detail'),
+    url(r'^(?P<pk>\w+)/confirm/(?P<token>\w+)$', views.NodeDeleteConfirm.as_view(), name='node-delete-confirm'),
```

Comment:
After speaking with BrianG, the inconsistency shouldn't be there - we should be terminating URLs with a slash. If there is a compelling reason to deviate from that standard we can re-evaluate at a late date.

As for the existing route that is already in place without a trailing slash, let's leave that be in this PR, and we'll address it separately.